### PR TITLE
Add migrations for fountains and reports entity

### PR DESCRIPTION
**Pull request by Vannnu**
Create tables for database entity.

### CODE REVIEW

**Comment by Vannnu** [comment]
@ptesser The migrations are already tested

**Comment by ptesser** [comment]
@Vannnu This branch has some merge commit of others branch that make the git story dirty. Use rebase strategy instead of merge. It's easier to do a review.

Rebase it and the push force on  it.

File: u-drink-water/database/migrations/2019_08_15_101735_create_table_fountains.php [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            $table->string('url_image', 255)
+                ->nullable()
+                ->comment('url image of the fountain');
+
+            $table->boolean('enable')
+                ->comment('true if the fountain is visible to user false otherwise');
+
+            $table->timestamps();
+
+            $table->primary(['lat', 'lng']);
```
Comment (ptesser) [inline u-drink-water/database/migrations/2019_08_15_101735_create_table_fountains.php]: Is `id` automatically a primary key?

File: php/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM php:7.0-fpm
```
Comment (ptesser) [inline php/Dockerfile]: This modify is not related with this pull request.

File: u-drink-water/database/migrations/2019_08_15_101735_create_table_fountains.php (hunk 1 above)
Comment (Vannnu) [reply u-drink-water/database/migrations/2019_08_15_101735_create_table_fountains.php]: $table->bigIncrements('id') yes
